### ISSUE TITLE
Autofix description show {{ correctedStoreName }} placeholder in WebStorm

### ISSUE THREAD

**Issue by SQReder** [effector/eslint-plugin#73]
rule: enforce-store-naming-convention
![image](https://user-images.githubusercontent.com/1519753/143016212-21909bd8-b660-4f41-9848-9b93d5d1cf07.png)

Not sure that it is really related to the plugin itself

**Comment by igorkamyshev** [comment]
Looks like a big 🤔

**Comment by SQReder** [comment]
Uh-oh!
```
    "eslint": "^7.32.0",
    "eslint-plugin-effector": "^0.4.0",
```

**Comment by igorkamyshev** [comment]
Will be fixed in 0.5.0
